client/index.html

```diff
@@ -6,6 +6,9 @@
     <link rel="stylesheet" type="text/css" href="style.css">
   </head>
   <body>
+    <a class="logo" href="http://www.moduscreate.com">
+    <svg viewBox="905 28 63 58" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M7.176 25.9C11.61 12.25 22.973 9.972 26.19 9.474c0 0 .323-4.938-1.808-9.34 0 0-10.22.994-17.72 10.26 0 0-6.53 6.66-6.53 19.045 0 6.18 2.02 11.23 5.464 16.4 3.407 5.11 11.972 12.08 22.63 12.08h.38v-5.03s-10.353-1.61-15.473-6.17c-7.328-6.51-6.7-18.57-5.957-20.83z" id="a"/><path d="M37.97 28.75c-3.12 1.552-6.127 2.107-8.81 2.107-5.635 0-9.917-2.45-11.25-3.35-9.08-6.125-11.058-16.596-7.84-24.77 0 0-3.313-1.837-8.243-2.565 0 0-3.768 9.456-.228 19.372 0 0 2.41 7.006 7.53 11.887 3.67 3.53 10.45 7.99 19.51 7.99 3.94 0 8.3-.84 13.03-2.95 5.65-2.52 16.06-11.2 16.29-25.34-5.12.64-8.95 3.43-8.95 3.43s-1.24 9.32-11.06 14.2z" id="c"/></defs><g fill="none" opacity=".299" fill-rule="evenodd"><g transform="translate(939.07 28)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path d="M-62.62-47.472V75.802H59.956V-47.472H-62.62zM27.73 11.16c-.056 3.503-.722 6.91-1.96 10.126-1.065 2.775-2.588 5.436-4.49 7.867-3.008 3.83-7.024 7.102-10.793 8.786C5.9 39.98 1.313 41.02-3.18 41.02c-4.3 0-8.487-.956-12.446-2.813-3.75-1.78-6.51-4.04-8.147-5.59-5.29-5.053-7.784-12.097-7.936-12.5-1.92-5.416-1.77-10.68-1.27-14.145.52-3.752 1.51-6.22 1.54-6.336l.46-1.167 1.22.19c5.14.74 8.62 2.66 8.78 2.75l1.22.67-.51 1.3c-2.76 7.06-1.595 16.88 7.27 22.85.766.51 4.858 3.08 10.377 3.08 2.78 0 5.5-.65 8.11-1.95 3.73-1.86 6.565-4.6 8.45-8.16 1.427-2.68 1.73-4.86 1.73-4.88l.077-.67.553-.406c.17-.115 4.204-3.024 9.67-3.694l1.785-.21v1.8z" fill="#1B1B1B" fill-rule="nonzero" mask="url(#b)"/></g><g transform="translate(907.284 28)"><mask id="d" fill="#fff"><use xlink:href="#c"/></mask><path d="M-30.834-47.472V75.802H91.74V-47.472H-30.834zm58.756 76.99c-.362 7.75-2.38 12.192-5.73 17.227-2.055 3.08-5.386 6.163-9.097 8.422C8.413 58.02 3.293 59.53-1.77 59.53h-.42l-1.56-.018v-8.135l1.523-.058c.076 0 8.375-.33 13.38-4.2 9.65-7.49 7.442-17.23 6.72-20.43-.058-.19-.077-.33-.115-.46-.438-2.09-2.97-7.62-8.032-11.39l-.495-.37-.11-.595c-.05-.25-1.23-6.24 1.09-11.083l.71-1.47 1.43.764c.08.06 2.15 1.13 4.7 3.294 2.348 1.99 5.6 5.32 7.71 9.86.31.594 3.488 7.025 3.18 14.26z" fill="#1B1B1B" fill-rule="nonzero" mask="url(#d)"/></g><path d="M918.894 31.503c-2.13 4.403-.97 10.07-.97 10.07 5.52 4.095 8.165 10.01 8.64 12.326.496 2.31 3.845 13.91-7.194 22.47-5.424 4.19-14.256 4.5-14.256 4.5v5.03h.38c10.66 0 19.243-6.97 22.63-12.08 3.446-5.17 5.14-9.32 5.464-16.41.323-7.08-2.988-13.48-2.988-13.48-3.845-8.33-11.706-12.44-11.706-12.44z" fill="#1B1B1B" fill-rule="nonzero"/></g></svg>
```

Comment:
please use external SVG file. We can demonstrate some pre-caching magic then ;)

Here's how

```jsx
import logo from 'images/mclogo.svg'; // make sure we have file-loader in webpack

<img src={logo} />
```

Webpack would optimally create a hash name and spit it out during build process